addons/html_editor/static/src/editor.js

```diff
@@ -179,10 +179,12 @@ export class Editor {
         }
     }
 
-    destroy() {
+    destroy(willBeRemoved) {
         if (this.editable) {
-            this.editable.removeAttribute("contenteditable");
-            removeClass(this.editable, "odoo-editor-editable");
+            if (!willBeRemoved) {
```

Comment:
Do we really need willBeRemoved?
When do we want to destroy without removing it?